src/Front/ParseOperators.hs

```diff
@@ -1,22 +1,24 @@
+{-# LANGUAGE OverloadedStrings #-}
 module Front.ParseOperators
   ( parseOperators
   , Fixity(..)
   ) where
 
 import qualified Common.Compiler               as Compiler
+import           Common.Identifiers
 
-import qualified Data.Map.Strict               as Map
 import           Front.Ast                      ( Definition(..)
                                                 , Expr(..)
+                                                , Fixity(..)
                                                 , OpRegion(..)
                                                 , Program(..)
                                                 )
 
-data Fixity = Infixl Int String
```

Comment:
This belongs in `Ast.hs` since eventually we will want to add fixity declarations etc. in the language